familyApp/app/components/Drawer.js

```diff
@@ -108,5 +109,6 @@ const styles = StyleSheet.create({
   },
 });
 /*
-  static router = MyDrawer.router;
+
```

Comment:
Unnecessary spaces

Reply:
it is only comment for me I will use it when I need (/* notes    */)